### PR TITLE
More Sendable updates

### PR DESCRIPTION
**Pull request by auscompgeek**
Update Encoder, AnalogGyro, DoubleSolenoid, and ADXRS450_Gyro for the LiveWindow changes.

### CODE REVIEW

**Comment by Twinters007** [comment]
Thought so. @ariovistus merge my PR so I can merge *your* PR so I can merge *this* PR

**Comment by auscompgeek** [comment]
Whoops, I forgot @ariovistus had an Encoder PR already open. Uh. One sec.

**Comment by auscompgeek** [comment]
@Twinters007 Could you please double-check that the DoubleSolenoid changes here are indeed correct?

File: wpilib/wpilib/doublesolenoid.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # validated: 2017-12-12 EN f9bece2ffbf7 edu/wpi/first/wpilibj/DoubleSolenoid.java
```
Comment (Twinters007) [inline wpilib/wpilib/doublesolenoid.py]: You didn't validate this

Comment (auscompgeek) [reply wpilib/wpilib/doublesolenoid.py]: It was already validated.

**Pull request merged**